README.md

```diff
@@ -1,5 +1,109 @@
+built, tested, and deployed.
+
+Over the duration of the course, you will build all the important parts of a
+web application that allows users to find and adopt adorable furry
+companions. Each workshop will focus on a different part of the application,
+and will draw on a different set of skills.
+
+Below, you'll find out what to expect in each workshop.
+
+## Course outline
+
+### Workshop 1: modern front-end web development
+
+In this workshop, we're going to build a frontend of a website for the pet
+adoption center. In doing this, you'll learn:
+
+- What is a single-page application (SPA)?
+- How to create a SPA using `create-react-app`.
+- How to scaffold a user interface using `react-bootstrap`.
+- What is an API (application programming interface)?
+- How to talk to an API from our React SPA.
+
+We'll also cover:
+
+- How to use the Git version control system to submit a pull request (PR).
+- How to use pull requests as part of a code review process.
+
+_If you're ready, go to the [course notes for Workshop 1](./week1-webclient.md)._
+
+### Workshop 2: APIs and back-end web development
+
+In the first workshop, you built a single-page application that talked to an
+API service that returned information about pets up for adoption. In this
+workshop, you're going to build the pet adoption API yourselves, and host it
+in the cloud. If things go according to plan, you'll learn:
+
+- How is an API application different from other web applications?
+- How to pick between different options for building and hosting web applications.
+- How to build an API for listing pets available for adoption.
+- How to host your API application in Azure.
+
+_The notes for Workshop 2 aren't quite ready yet. We'll add them here when
+they are._
+
+### Workshop 3: Testing, Continuous Integration, Continuous Deployment
+
+So far, you've built both a front-end and a back-end for the pet adoption
+center's website. In this workshop, we're going to look at how you can test
+the changes you make to your software, and how you can integrate those tests
+into a pipeline to help you release software quickly and easily. We hope
+you'll learn:
+
+- How to write tests for your code.
+- What kinds of tests can you write, and what is the "testing pyramid"?
+- How to run tests automatically after every change.
+- How to use tests to provide automatic code review.
+- How to automatically deploy changes to your code after they pass the tests.
+
+_The notes for Workshop 3 aren't quite ready yet. We'll add them here when
+they are._
+
+### Workshop 4: Logs, Metrics, and Observability
+
+You have now built, tested, and deployed a web application to the cloud! Now
+you want to know whether it's doing the right thing. In this workshop, we're
+going to look at how to use logs and metrics to understand what your
+application is really doing in production. By adding instrumentation to your own application, you'll learn:
```

Comment:
This all sounds great, but the SRE in me wants to sell the idea of "We'd like to make sure that your **customers** are able to happily use your service". What do you think?

Reply:
Yup. I think for the pet shelter it may make sense to talk about "users" rather than customers, but I'll definitely reword this to frame that as the overall goal.